docs/add-ons/istio.md

```diff
@@ -0,0 +1,33 @@
+# Istio
+
+[Istio](https://istio.io/) is an open source service mesh that layers transparently onto existing distributed applications. Istio’s powerful features provide a uniform and more efficient way to secure, connect, and monitor services.
```

Comment:
Lets provide more actionable information - if someone is enabling Istio, they most likely already know what it is so this current statement doesn't provide much value

For example - theres 4 components within this module - whats the relationships, what are the different combinations and why, what does each combination give users, what is the bare minimum MVP case and whats advanced? What about the other charts in the istio repository https://github.com/istio/istio/tree/master/manifests/charts?